koku/masu/processor/tasks.py

```diff
@@ -654,7 +654,8 @@ def update_summary_tables(  # noqa: C901
     if not manifest_list and manifest_id:
         manifest_list = [manifest_id]
 
-    if cost_model is not None:
+    # OCP cost distribution of unattributed costs should happen even without a cost model
```

Comment:
This comment isn't super clear:
```suggestion
    # OCP cost distribution of unattributed costs occurs within the `update_cost_model_costs` method. This method should always be called for OCP providers even when it does not have a cost model
```